### ISSUE TITLE
Does the object detection pre-training dataset include val/test set in original datasets?

### ISSUE THREAD

**Issue by waxnkw** [pzzhang/VinVL#17]
The pre-training dataset is combined with several object detection datasets e.g.: VG, COCO.  Are the original val/test set of VG and COCO included?
